scripts/utils/default_yargs.js

```diff
@@ -10,3 +10,18 @@ module.exports = require("yargs")
     describe: "network where the script is executed",
     choices: ["rinkeby", "mainnet"],
   })
+
+const checkNoDuplicate = function (array) {
+  return new Set(array).size !== array.length
+}
+
+function checkBracketsForDuplicate(argv) {
+  if (argv.brackets && checkNoDuplicate(argv.brackets))
+    throw new Error("the parameter --brackets is not supposed to have duplicated entries")
+  return true
+}
```

Comment:
This is looking a lot better, but wonder if we could get away without a dedicated single use function like `checkBracketsForDuplicate`. For example, the more general function could 

```js 
return array && new Set(array).size !== array.length
```

but then the error message would have to go up in the `.check(checkBracketsForDuplicate).argv` I would imagine that `.check` since it is apparently a feature of yargs already has a way to include custom error message upon failure.

Reply:
I checked the documentation and could not find a custom error message for .check

See also here for a search:
https://stackoverflow.com/questions/38419991/yargs-check-error-handling

Reply:
Also, I don't wanna write the same error msg into all script files, which have the parameter brackets